src/RMPI/onesided.jl

```diff
@@ -10,11 +10,12 @@ function mpi_one_sided(data, comm = mpi_comm(), root = mpi_root)
     MPI.Initialized() || error("MPI needs to be initialised first.")
     np = MPI.Comm_size(comm)
     id = MPI.Comm_rank(comm)
-    P = pairtype(data)
+    P = eltype(data)
```

Comment:
I think this does not work as expected right now with initiators. When setting up `MPIData` with `mpi_one_sided` and `InitiatorDVec`, it seems like the `InitiatorValue` type information gets lost and the `MPI.Win` and associated buffer are created with the wrong value type.

Reply:
Probably this should read
```suggestion
    P = eltype(storage(data))
```
but I am not sure whether other code changes are required.